mlflow/R/mlflow/DESCRIPTION

```diff
@@ -55,11 +55,12 @@ Suggests:
     h2o,
     keras,
     lintr,
-    mleap,
     sparklyr,
     stringi (< 1.4.4),
     testthat (>= 2.0.0),
     xgboost
+Enhances:
+    mleap
```

Comment:
Without this section, `R CMD check` would complain:

https://github.com/mlflow/mlflow/runs/4514454297?check_suite_focus=true#step:14:197

```
── R CMD check results ────────────────────────────────────── mlflow 1.22.1 ────
Duration: 42s

❯ checking dependencies in R code ... WARNING
Warning:   '::' or ':::' import not declared from: ‘mleap’
```

Reply:
https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-Dependencies explains what the `Enhances` field means:

> Finally, the ‘Enhances’ field lists packages “enhanced” by the package at hand, e.g., by providing methods for classes from these packages, or ways to handle objects from these packages (so several packages have ‘Enhances: chron’ because they can handle datetime objects from chron even though they prefer R’s native datetime functions). Version requirements can be specified, but are currently not used. Such packages cannot be required to check the package: any tests which use them must be conditional on the presence of the package. (If your tests use e.g. a dataset from another package it should be in ‘Suggests’ and not ‘Enhances’.)